### PR TITLE
New structure of the project. All code modules under module folder

### PR DESCRIPTION
**Pull request by rusboy**
1. All module specific configs and files have been moved under module folder
2. styles.css from app folder has been moved under Styles folder
3. "Default" folder structure for each module

FYI - Seems redux counter is brokern, it's good next task

### CODE REVIEW

File: frontend/src/app/modules/login/containers/login-screen/login-screen.component.spec.ts [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
-  let component: LoginScreenComponent;
-  let fixture: ComponentFixture<LoginScreenComponent>;
+describe('LoginScreenContainerComponent', () => {
+  let component: LoginScreenContaineromponent;
+  let fixture: ComponentFixture<LoginScreenContaineromponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ LoginScreenComponent ]
+      declarations: [ LoginScreenContaineromponent ]
```
Comment (PMashaI) [inline frontend/src/app/modules/login/containers/login-screen/login-screen.component.spec.ts]: Seems you want to named it LoginScreenContainerComponent

File: frontend/src/app/modules/login/containers/login-screen/login-screen.component.ts [hunk 2]
```diff
@@ -29,6 +32,14 @@ export class LoginScreenComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    this.subscription = this.count$.subscribe();
+    console.log(this.subscription);
```
Comment (PMashaI) [inline frontend/src/app/modules/login/containers/login-screen/login-screen.component.ts]: Do we need to track something here?

**Review by PMashaI — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/login/containers/login-screen/login-screen.component.ts (hunk 2 above)
Comment (rusboy) [reply frontend/src/app/modules/login/containers/login-screen/login-screen.component.ts]: removed. Please fix with new structure existed redux scenario (it's good case to start working with ngrx)

File: frontend/src/app/modules/login/containers/login-screen/login-screen.component.spec.ts (hunk 1 above)
Comment (rusboy) [reply frontend/src/app/modules/login/containers/login-screen/login-screen.component.spec.ts]: yes, it was renamed. 
Container -> it's SMART  -> It's top level component of module that uses another components/directives
Comonent -> It's Dump -> aka directive

File: frontend/src/app/modules/login/containers/login-screen/login-screen.component.ts (hunk 2 above)
Comment (PMashaI) [reply frontend/src/app/modules/login/containers/login-screen/login-screen.component.ts]: It is not working right now with the new structure, rigth?

**Pull request merged**